src/components/pages/DashboardPage/DashboardPage.js

```diff
@@ -10,15 +10,17 @@ export default class DashboardPage extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      selectedNodes: this.props.location.state,
+      selectedNodes: this.props.location.state.selectedNodes,
+      startDate: this.props.location.state.startDate,
+      endDate: this.props.location.state.endDate,
     };
   }
 
   render() {
     return (
       <div >
         <TopNavigationBar />
-        <DashboardPanel selectedNodes={this.state.selectedNodes} />
+        <DashboardPanel selectedNodes={this.state.selectedNodes} startDate={this.state.startDate} endDate={this.state.endDate} />
```

Comment:
Could try deconstructing state at the start of the render method 👍 probably best to do this once we start calling `this.state` on >=3 objects